docs/data/osdf/install-cache-rpm.md

```diff
@@ -0,0 +1,171 @@
+
+* __Operating system:__ A RHEL 8 or RHEL 9 or compatible operating systems.
+* __User IDs:__ If they do not exist already, the installation will create the Linux user named `xrootd` for running daemons.
+* __File Systems:__ The cache should have a partition of its own for storing data and metadata.
+* __Host certificate:__ Required for authentication.  See note below.
+* __Network ports:__ The cache service requires the following ports open:
+  * Inbound TCP port 8443 for file access via the HTTP(S) and XRoot protocols.
+  * (Optional) Inbound TCP port 8444 for access to the web interface for monitoring and configuration;
+    if enabled, this should be restricted to the LAN.
+* __Service requirements:__
+    * A cache serving the OSDF federation as a regional cache should have at least:
+        * 8 cores
+        * 40 Gbps connectivity
+        * 50-200 TB of NVMe disk for the cache partition; you may distribute the disk, e.g., by using an NVMe-backed Ceph pool,
+            if you cannot fit that much disk into a single chassis
+        * 24 GB of RAM
+    * A cache being used to serve data from the OSDF to a single site should have at least:
+        * 8 cores
+        * 40 Gbps connectivity
+        * 2 TB of NVMe disk for the cache partition
+        * 24 GB of RAM
+  We suggest that several gigabytes of local disk space be available for log files,
+  although some logging verbosity can be reduced.
+
+As with all OSG software installations, there are some one-time steps to prepare in advance:
+
+* Obtain root access to the host
+* Prepare [the required Yum repositories](../../common/yum.md)
+
+
+!!! note "OSG 23"
+    In OSG 23, the Pelican-based OSDF RPMs are only available in the "osg-upcoming" repositories.
+
+!!! note "Host certificates"
+    Caches should use a CA that is accepted by major browsers and operating systems,
+    such as InCommon RSA or [Let's Encrypt](../../security/host-certs/lets-encrypt).
+    IGTF certs are not recommended because clients are not configured to accept them by default.
+    Note that you will need the full certificate chain, not just the certificate.
+
+    The following locations should be used (note that they are in separate directories):
+    
+    * **Host Certificate Chain**: `/etc/pki/tls/certs/pelican.crt`
+    * **Host Key**: `/etc/pki/tls/private/pelican.key`
+
+
+Installing the Cache
+--------------------
+
+The cache service is provided by the `osdf-cache` RPM.
+Install it using one of the following commands:
+
+OSG 23:
+```console
+root@host # yum install --enablerepo=osg-upcoming osdf-cache
+```
+
+OSG 24:
+```console
+root@host # yum install osdf-cache
+```
+
+
+Configuring the Cache Server
+----------------------------
+
+!!! note "osdf-cache 7.11.1"
+    This configuration requires version 7.11.1 or newer of the `osdf-cache`
+    and `pelican` RPMs.
+    
+Configuration for a Pelican-based OSDF Cache is located in files in `/etc/pelican/config.d`.
```

Comment:
Maybe we throw this in a table in the appendix like we used to do? Something like:

```
| **Location**                   | **Description**                                                                                                                                                                 |
|--------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `/etc/pelican/config.d/`       | Administrator controlled directory for OSDF cache configuration. `*.conf` files in this directory are parsed in lexicographic order. Changes are preserved upon package update. |
| `/etc/pelican/osdf-cache.yaml` | OSDF cache configuration entrypoint. Changes are overridden upon package update.                                                                                                     |
| `/usr/local/pelican/config.d`  | Packaged configuration defaults. Changes overridden upon package update.                                                                                                         |
```

Reply:
I think that's too much info.  Admins should touch `/etc/pelican/config.d` and leave everything else alone; how we add our config is an implementation detail.

Reply:
It could be useful for troubleshooting 🤷🏽 but I'm fine excluding it from this PR. Without the additional detail, though, I think the sentence is of questionable use since we tell them which configs to edit.

Reply:
Fair enough; I'll remove that sentence.

(For troubleshooting, I think we're going to need that cconfig-esque tool -- we change so few of the knobs that just reading the files on disk won't give you a good idea of what everything's set to.)

Reply:
Yeah that'll be great but in the meantime, there's the startup banner that dumps the config into the logs